simpleRepository/MemoryRepository.test.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * Unit test for MemoryRepository and hgrs.
```

Comment:
No need to mention `hgrs`. This test is just for MemoryRepository.

Although same tests may be partially used to create a system test for `UniversalRepository` later.